front/post_upsert_hooks/temporal/activities.ts

```diff
@@ -46,6 +46,41 @@ export async function runPostUpsertHookActivity(
   localLogger.info("Ran post upsert hook function.");
 }
 
+export async function runOnDeleteActivity(
```

Comment:
NIT: shouldn't it be `runPostDeleteHookActivity` for consistency?  (replacing "onDelete" by "PostDeleteHook" everywhere in names)

Reply:
Agreed, I think `PostUpsertHook` needs a rename. Some candidates:
- `DocumentPostEditHook`
- `DocumentLifecycleHook`

I don't love either of them, curious if you have others ?

Reply:
I suggested IRL `DocumentPostProcessHook`, a bit more generic but covers the create/edit/delete cases 🙇🏻‍♀️